AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -622,9 +622,8 @@ extension MeetingModel: ActiveSpeakerObserver {
     }
 
     func activeSpeakerDidDetect(attendeeInfo: [AttendeeInfo]) {
-        videoModel.updateRemoteVideoStatesBasedOnActiveSpeakers(activeSpeakers: attendeeInfo)
         if activeMode == .video {
-            videoModel.videoUpdatedHandler?()
+            videoModel.updateRemoteVideoStatesBasedOnActiveSpeakers(activeSpeakers: attendeeInfo)
```

Comment:
If we move this statement here, we would not be able to handle the case where active speaker changed while the app is on other tabs, right?

Reply:
Right, I can make another revision with
```
videoModel.updateRemoteVideoStatesBasedOnActiveSpeakers(activeSpeakers: attendeeInfo)
if activeMode == .video && videoModel.videoTilesOrderUpdated {
    videoModel.videoUpdatedHandler?()
}
```
We just need to add a public boolean to videoModel